app/application/-components/verify-email-modal/component.ts

```diff
@@ -115,18 +115,17 @@ export default class VerifyEmailModal extends Component.extend({
                 verifyError: 'verifyEmail.add.verifyError',
                 denyError: 'verifyEmail.add.denyError',
             };
-        } else {
-            return {
-                header: 'verifyEmail.merge.header',
-                body: 'verifyEmail.merge.body',
-                verifyButton: 'verifyEmail.merge.verifyButton',
-                denyButton: 'verifyEmail.merge.denyButton',
-                verifySuccess: 'verifyEmail.merge.verifySuccess',
-                denySuccess: 'verifyEmail.merge.denySuccess',
-                verifyError: 'verifyEmail.merge.verifyError',
-                denyError: 'verifyEmail.merge.denyError',
-            };
         }
+        return {
```

Comment:
Is this a stylistic requirement or a refactor?

Reply:
stylistic requirement: [no-else-return](https://eslint.org/docs/rules/no-else-return)

[eslint-config-airbnb-base@12.1.0](https://github.com/airbnb/javascript/blob/eslint-config-airbnb-base-v12.1.0/packages/eslint-config-airbnb-base/rules/best-practices.js#L62) had `allowElseIf: true` (see TODO note about changing this)

True to their word, as of [eslint-config-airbnb-base@13.0.0](https://github.com/airbnb/javascript/blob/eslint-config-airbnb-base-v13.0.0/packages/eslint-config-airbnb-base/rules/best-practices.js#L61) has `allowElseIf: false`

Reply:
Yea I saw it in a couple of places (I think I commented below). Sorry PR was more of a stream of consciousness - but I remember this being enforced in newer versions of eslint.